include/ListView/ListViewController.php

```diff
@@ -208,8 +232,11 @@ function getListViewEntries($focus, $module,$result,$navigationInfo,$skipActions
 					if (is_null($field)) continue;
 				}
 				$uitype = $field->getUIType();
-				$rawValue = $this->db->query_result($result, $i, $field->getColumnName());
-				if($module == 'Calendar') {
```

Comment:
I think you have made a mistake here. You have to leave the

```
if($module=='Calendar')
```

and put it below with the $activityType assignment